awx/api/serializers.py

```diff
@@ -4614,6 +4614,23 @@ def get_related(self, obj):
             res['inventory'] = obj.unified_job_template.inventory.get_absolute_url(self.context.get('request'))
         return res
 
+    def get_summary_fields(self, obj):
+        summary_fields = super(ScheduleSerializer, self).get_summary_fields(obj)
+        if 'inventory' in summary_fields:
+            return summary_fields
+
+        inventory = None
+        if obj.unified_job_template and getattr(obj.unified_job_template, 'inventory', None):
+            inventory = obj.unified_job_template.inventory
+        else:
+            return summary_fields
+
+        summary_fields['inventory'] = dict()
```

Comment:
is it worth it to *check* that inventory already exists in summary fields and then not go through this process if it does?

Otherwise this looks good and will attach what we expect to the summary fields.